packages/app/README.md

```diff
@@ -286,16 +286,13 @@ specific export scenarios. In this case, or if you don't provide a function at
 all, `H5GroveProvider` falls back to generating URLs based on the `/data`
 endpoint and `format` query param.
 
-#### `key?: Key` (optional)
+#### `resetKeys?: unknown[]` (optional)
 
-If the content of the current file changes and you want to ensure that the
-viewer refetches the latest metadata and dataset values, you can take advantage
-of
-[React's `key` attribute](https://react.dev/reference/react/useState#resetting-state-with-a-key).
-Changing the value of the `key` will force a remount of `H5GroveProvider` and
-clear its internal fetching cache.
+Use this prop to reset the provider's internal fetch cache, for instance when
+the content of the current file changes and you want the viewer to refetch the
+latest metadata and dataset values.
```

Comment:
I think we should keep the link to the React doc that explains what a key is.

Or explain a bit more how to use this prop:

> In `resetKeys`, you can put variables that when changed, will reset the provider's internal fetch cache.

Reply:
Good call and good formulation :+1: 

I'm not adding a link to the React doc, as I feel it could create confusion.